coral/src/domain/acl/acl-api.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  CreateAclRequestTopicTypeProducer,
+  CreateAclRequestTopicTypeConsumer,
+} from "src/domain/acl/acl-types";
+import api from "src/services/api";
+import { KlawApiResponse } from "types/utils";
+
+const postAclRequest = (
```

Comment:
I would suggest we rename this into something like "createAclRequest". I think the HTTP method `post` is a implementation detail we want to abstract away, so I would not mention it in the function name.

Reply:
Indeed, makes sense.